scripts/pod_lib_lint.rb

```diff
@@ -52,9 +53,13 @@ def main(args)
   # to the pod command.
   pod_args = []
   ignore_local_podspecs = []
+  skip_analyze = false
```

Comment:
Optional: since the point of this is to compute the `--analyze` flag we're passing down to `pod lib lint`, you might make this `analyze = true` to avoid the double negative below (the "not skip").

Reply:
Done, thanks.